core/lib/dal/src/blocks_dal.rs

```diff
@@ -450,6 +450,7 @@ impl BlocksDal<'_, '_> {
         predicted_block_gas: BlockGasCount,
         events_queue: &[LogQuery],
         storage_refunds: &[u32],
+        predicted_circuits: u32,
```

Comment:
where is a good place to add the description on what 'predicted_circuits' mean?

Reply:
I think adding a comment to this method declaration will work best, will add it with the next PR I open